monkey/monkey_island/cc/ui/src/components/configuration-components/PluginSelectorTemplate.tsx

```diff
@@ -8,12 +8,14 @@ import {InfoPane, WarningType} from '../ui-components/InfoPane';
 import {EXPLOITERS_PATH_PROPAGATION} from './PropagationConfig';
 import MarkdownDescriptionTemplate from './MarkdownDescriptionTemplate';
 
+const pluginSchemaPath = {'propagation': EXPLOITERS_PATH_PROPAGATION, 'credentials_collectors': 'credentials_collectors'}
+
```

Comment:
This is inconsistent. If we have constants for the exploiters configuration path in the file relevant to them, we should have them for credential collectors too.